onboarding/src/Components/utils.js

```diff
@@ -61,3 +60,9 @@ export function validEmail(email){
 	return regExp.test(String(email));
 }
 
+export function clickButtonAfterPressingEnter(e, buttonId) {
+	if (e.key === "Enter") {
```

Comment:
Another common statement
`if(e.keyCode == 13)`
So what would you say about
`if(e.key === "Enter" || e.keyCode == 13)`?